lib/fathead/arrow_docs/parse.py

```diff
@@ -37,8 +37,11 @@ def yield_api_example(api_examples):
 
         abstract = clean_output(api_example.dd.p.text)
         abstract = abstract_fmt.format(abstract, code)
+        
+        # redirect_alias: => 'arrow.arrow.Arrow.clone' => arrow clone
+        redirect_alias = 'arrow ' + title.split('.')[-1]
 
-        yield [title, 'A', '', '', '', '', '', '', '', '', '', abstract, url]
```

Comment:
I'm a bit confused @moollaza should i add redirects for each article in `output.txt` below and leave the existing Articles as they are (Doesn't make much sense to me) 

Or should i use the `redirects.txt` file to add the redirects?